### ISSUE TITLE
TF debug crashes when nan in loss

### ISSUE THREAD

**Issue by kesinger** [tensorflow/tensorflow#25395]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No, beyond the trivial fix needed for  https://github.com/tensorflow/models/issues/5857, and wrapping the tf.Session in a debug session
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): Binary
- TensorFlow version (use command below): v1.12.0-0-ga6d8ffae09 1.12.0
- Python version: 3.6.7
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: 9.0
- GPU model and memory: 1080ti / 11GB

**Describe the current behavior**
When run in the debugger, if there is a NaN in the loss, the debugger crashes.

**Describe the expected behavior**

Expected behavior is that the debugger does not crash and I can examine the tensors.
**Code to reproduce the issue**

In models/research/struct2depth, run: 
ckpt_dir="ckpt"
data_dir="/opt/baddata/tfrep"

python train.py \
  --logtostderr \
  --checkpoint_dir=$ckpt_dir  \
  --data_dir=$data_dir \
  --architecture=resnet \
  --imagenet_norm=true \
  --joint_encoder=false

**Other info / logs**

I'm attaching a screenshot of the terminal after tfdbg crashes (the terminal gets corrupted so I can't paste this into a text file) :  
<img width="1422" alt="screen shot 2019-01-28 at 12 06 26" src="https://user-images.githubusercontent.com/4651708/52070300-8628ac00-254e-11e9-835e-e5ff4ff84282.png">

A set of input data that triggers this crash will be uploaded soon.

**Comment by kesinger** [comment]
I'm attaching a tgz file with a set of ten images that will trigger this crash.  They're originally from the kitti dataset.   The "train.txt" that corresponds to this looks something like:

/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_03 0000001706
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_03 0000000312
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_02 0000002585
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_03 0000001151
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_02 0000000365
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_03 0000000829
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_03 0000000092
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_02 0000001900
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_03 0000002574
/opt/bigdata/kitti_processed/2011_09_30_drive_0018_sync_02 0000000327

[badkitti.tar.gz](https://github.com/tensorflow/tensorflow/files/2818949/badkitti.tar.gz)

**Comment by jvishnuvardhan** [comment]
@kesinger TF1.x is deprecated. Please update to TF2.x debugger and open a new issue with a simple standalone code to reproduce the issue.

I am closing this as the TF1.x is deprecated. Please feel free to reopen I am mistaken. Thanks!
